### DISCUSSION TITLE
Get cookie from response set-cookie header - Flutter web , Getconnect

### DISCUSSION THREAD

**Discussion by bb-renittojose** [jonataslaw/getx#2508] [Q&A]
I'm getting the cookie on the response headers and I want to extract that token data, I want to get the value of **csurftoken** from the response header **set-cookie** on flutter web. I'm using Getconnect for networking.

I tried to get the cookies using the following but got empty only

```
if (response.headers == null ||           !response.headers!.containsKey(HttpHeaders.setCookieHeader)) log('setCookie is empty');
        var setCookie = response.headers![HttpHeaders.setCookieHeader];
        log('setCookie Data: $setCookie');
```

<img width="508" alt="image" src="https://user-images.githubusercontent.com/106244321/184880397-85d4dca4-a0d2-4481-8c3e-affb3657bb81.png">

**Comment by dominicmh** [comment]
Browsers block certain headers (e.g., `Authorization`, `Set-Cookie`) under specific CORS rules. So you won't be able to read them from Flutter.
